### ISSUE TITLE
[~2.3.64] Wrong Database Prefix in Enterprise Class

### ISSUE THREAD

**Issue by Pacolu** [akeneo/pim-community-dev#10955]
**I'm reporting a Bug**
I cannot do this: "System > system info", since the install process fails.

During the install process of our project, but could confirm this by using the default icecat demo, while importing the fixtures, the process crashes. The Exception is this:

```
Nothing to update - your database is already in sync with the current entity metadata.
Reset elasticsearch indexes
Create session table
Load jobs for fixtures. (data set: PimInstallerBundle:icecat_demo_dev)
Fixtures fixtures_currency_csv has been successfully executed.
Fixtures fixtures_locale_csv has been successfully executed.
Fixtures fixtures_user_group_csv has been successfully executed.
Fixtures fixtures_category_csv has been successfully executed.
Fixtures fixtures_attribute_group_csv has been successfully executed.
Fixtures fixtures_group_type_csv has been successfully executed.
Fixtures fixtures_attribute_csv has been successfully executed.
10:29:44 WARNING   [translation] Translation not found. ["id" => """  An exception occurred while executing 'SELECT DISTINCT locale.code\n  FROM pim_catalog_channel AS channel\n    INNER JOIN akeneo_pim.pim_catalog_channel_locale AS channel_locale ON (channel.id = channel_locale.channel_id)\n    INNER JOIN akeneo_pim.pim_catalog_locale AS locale ON (channel_locale.locale_id = locale.id)\n  WHERE channel.code = ?' with params ["print"]:\n  \n  SQLSTATE[42000]: Syntax error or access violation: 1142 SELECT command denied to user 'akeneo'@'IP' for table 'pim_catalog_channel_locale'  ""","domain" => "messages","locale" => "en"]
10:29:44 ERROR     [batch] Encountered an error executing the step: An exception occurred while executing 'SELECT DISTINCT locale.code
FROM pim_catalog_channel AS channel
  INNER JOIN akeneo_pim.pim_catalog_channel_locale AS channel_locale ON (channel.id = channel_locale.channel_id)
  INNER JOIN akeneo_pim.pim_catalog_locale AS locale ON (channel_locale.locale_id = locale.id)
WHERE channel.code = ?' with params ["print"]:

SQLSTATE[42000]: Syntax error or access violation: 1142 SELECT command denied to user 'akeneo'@'IP' for table 'pim_catalog_channel_locale'

SQLSTATE[42000]: Syntax error or access violation: 1142 SELECT command denied to user 'akeneo'@'IP' for table 'pim_catalog_channel_locale'
An error occurred during the fixtures execution.
Error #0 in class Doctrine\DBAL\Exception\ConnectionException: An exception occurred while executing 'SELECT DISTINCT locale.code
FROM pim_catalog_channel AS channel
  INNER JOIN akeneo_pim.pim_catalog_channel_locale AS channel_locale ON (channel.id = channel_locale.channel_id)
  INNER JOIN akeneo_pim.pim_catalog_locale AS locale ON (channel_locale.locale_id = locale.id)
WHERE channel.code = ?' with params ["print"]:
```

The reason is, that for Akeneo 2.3.64 this file exists:
https://github.com/akeneo/pim-community-dev/blob/v2.3.68/src/Pim/Bundle/CatalogBundle/Doctrine/ORM/Query/GetChannelActiveLocaleCodes.php

According to its own header, this is an enterprise class. Within this class, it uses the database prefix akeneo_pim. which is wrong, unless you stay with the default values within the parameters.yml.dist.

Strangely this error only occurs, if you have 2 or more channels in your channels.csv within your fixtures. If you use one, it runs through.

Steps to reproduce it:
- Download/Prepare an Akeneo Instance with ^2.3.64
- Run: bin/console pim:installer:db

**Comment by jmleroux** [comment]
Hello,

Thank you very much for raising this issue.
You right with the database prefix. It is not even related to Enterprise Edition. Bad copy-paste imho.
We didn't see it because our tests are done with these default values. Tests can be fooled sometimes :wink: .
The fix PR is already launched on our CI.

**Comment by Pacolu** [comment]
Ah ok. Yeah thats true.
Thank you for fixing it so fast :D

**Comment by jmleroux** [comment]
Fixed in v2.3.69
